CMakeLists.txt

```diff
@@ -1247,16 +1247,9 @@ if (ENABLE_EXAMPLES)
 
 	srt_add_example(recvlive.cpp)
 
-# Don't compile examples that do require C++11
-
-	if (ENABLE_CXX11)
-		srt_add_example(sendfile.cpp apps/logsupport.cpp)
-		target_compile_options(sendfile PRIVATE ${CFLAGS_CXX_STANDARD})
-        
-		srt_add_example(recvfile.cpp apps/logsupport.cpp)
-		target_compile_options(recvfile PRIVATE ${CFLAGS_CXX_STANDARD})
-	endif()
-
+	srt_add_example(sendfile.cpp)
+		
+	srt_add_example(recvfile.cpp)
```

Comment:
If you change the parameter for `srt_setloglevel` to `LOG_DEBUG`. The logger interface may require C++11, but it's not required by the SRT API.

Reply:
Hm, weird if it does.
Anyway, just removed the `srt_setloglevel`. It is out of the scope of those examples.